docs/Actions.md

```diff
@@ -641,6 +641,8 @@ The uploaded `version.json` file provides an easy way for apps to poll if a new
 }
 ```
 
+This action currently supports [aws-sdk v2](https://github.com/aws/aws-sdk-ruby)
```

Comment:
Hi,  I am not sure where to ask this.  I have been dealing with this S3 problem for the last few days.  no matter what I try it doesn't work.  I came across this solution so I forked the fastlane github and applied this fix.  I still get the shenzen issue and any help would be appreciated:  
[15:33:16]: Exit status of command 'krausefx-ipa distribute:s3  <----- removed rest of command ------>
error: uninitialized constant Shenzhen::Plugins::S3::Client::AWS. Use --trace to view backtrace